sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -613,6 +608,62 @@ def set_scalar(accumulator, metric_update):
   metric_update.scalar = to_json_value(accumulator.value, with_type=True)
 
 
+class CounterUpdateSetters(object):
+  """Setters for translating between accumulators and dataflow counters."""
```

Comment:
It's a bit unclear because we use the language interchangeably - and not so much interchangeably in Dataflow. In the work item protobuf, there are two lists of items: `counterUpdates` and `metricUpdates`. They both go into the dataflow service, and are converted into *dataflow counters*, which is an internal dataflow concept that keeps coutners such as `mean` and `sum`, but also distributions and progress updates.

I have renamed every setter method to `translate_*` equivalents, and renamed the setters class to be more clear.